src/ducks/future/PlannedTransactionsPage.jsx

```diff
@@ -0,0 +1,138 @@
+
+const InvertedCard = withStyles({
+  card: {
+    backgroundColor: 'var(--primaryColorLight)',
+    border: 0
+  }
+})(({ classes, children }) => {
+  return <Card className={classes.card}>{children}</Card>
+})
+
+const HeaderCard = () => {
+  const { t } = useI18n()
+  const { estimatedBalance, currency, transactions } = useEstimatedBudget()
+
+  if (estimatedBalance === null) {
+    return null
+  }
+
+  return (
+    <InvertedCard>
+      <Media>
+        <Bd>
+          <Typography variant="h6" color="primary">
+            {t('EstimatedBudget.30-day-balance')}
+          </Typography>
+          <Typography variant="caption" color="textSecondary">
+            {t('EstimatedBudget.numberEstimatedTransactions', {
+              smart_count: transactions.length
+            })}
+          </Typography>
+        </Bd>
+        <Img>
+          <Figure
+            className="u-ml-2"
+            total={estimatedBalance}
+            symbol={getCurrencySymbol(currency)}
+          />
+        </Img>
+      </Media>
+    </InvertedCard>
+  )
+}
+
+const headerStyle = {
+  zIndex: 'var(--z-index-bar)'
+}
+
+const listMobileStyle = {
+  marginTop: '6.5rem'
+}
+
+const PlannedTransactionsPage = () => {
+  const budget = useEstimatedBudget()
+  const router = useRouter()
+  const { t } = useI18n()
+  const { isMobile } = useBreakpoints()
+  const handleBack = useCallback(() => {
+    router.push('/balances/details')
+  }, [router])
+  return (
+    <>
+      <Header theme="inverted" fixed style={headerStyle}>
+        <BarTheme theme="primary" />
+        <Padded>
+          {isMobile ? (
+            <div
+              className="u-stack-s"
+              style={{
+                marginTop: '-3.25rem',
+                zIndex: 50,
+                position: 'relative'
+              }}
+            >
+              <BackButton theme="primary" onClick={handleBack} />
+              <div className="u-ta-center u-mh-1">
+                <Typography color="primary" variant="h5">
+                  {t('EstimatedBudget.page-title')}
+                </Typography>
+                <AccountSwitch size="small" insideBar={false} />
+              </div>
+              <HeaderCard />
+            </div>
+          ) : (
+            <Media>
+              <Img>
+                <BackButton arrow onClick={handleBack} theme="primary" />
+              </Img>
+              <Bd className="u-stack-xs">
+                <AccountSwitch size="normal" />
+                <div>
+                  <Typography color="primary" variant="h3">
+                    {t('EstimatedBudget.page-title')}
+                  </Typography>
+                </div>
+              </Bd>
+              <Img>
+                <HeaderCard />
+              </Img>
+            </Media>
+          )}
+        </Padded>
+      </Header>
+      <div
+        style={isMobile ? listMobileStyle : null}
+        className="js-scrolling-element"
```

Comment:
curious, why this className? What lib do you have to manage scrolls?

Reply:
Hmm.. a custom one done in Banks and working not so great.